vignettes/bettr.Rmd

```diff
@@ -159,11 +188,11 @@ results for the example above.
 #| fig.width: 7
 #| fig.height: 7
 
+## Assign a higher weight to one of the collapsed metric classes
+metadata(duo2018)$bettrInfo$initialWeights["Class_ARI"] <- 0.55
+
```

Comment:
I like this one. Goes "enough in depth" without becoming too complex!

Did we specify anywhere how the weight values were?